doc/md/tutorial/_helm-values/examples/02-helm-values.txt

```diff
@@ -172,49 +175,51 @@ Platform: Components: {
 }
 -- render.sh --
 holos render platform
--- register.components.output.txt --
+-- register-components-output.txt --
 cached prometheus-blackbox-exporter 9.0.1
 rendered blackbox in 3.825430417s
 cached prometheus 25.27.0
 rendered prometheus in 4.840089667s
 rendered platform in 4.840137792s
--- register.components.git.commit.sh --
+-- register-components-git-commit.sh --
 git add . && git commit -m 'add blackbox and prometheus'
--- register.components.git.commit.output.txt --
+-- register-components-git-commit-output.txt --
 [main b5df111] add blackbox and prometheus
  5 files changed, 1550 insertions(+)
  create mode 100644 components/blackbox/blackbox.cue
  create mode 100644 components/prometheus/prometheus.cue
  create mode 100644 deploy/components/blackbox/blackbox.gen.yaml
  create mode 100644 deploy/components/prometheus/prometheus.gen.yaml
  create mode 100644 platform/prometheus.cue
--- import.prometheus.values.sh --
+-- import-prometheus-values.sh --
 holos cue import \
   --package holos \
   --path 'Helm: Values:' \
   --outfile components/prometheus/values.cue \
   components/prometheus/vendor/25.27.0/prometheus/values.yaml
--- import.blackbox.values.sh --
+-- import-blackbox-values.sh --
 holos cue import \
   --package holos \
   --path 'Helm: Values:' \
   --outfile components/blackbox/values.cue \
   components/blackbox/vendor/9.0.1/prometheus-blackbox-exporter/values.yaml
--- import.values.render.output.txt --
+-- import-values-render-output.txt --
 rendered blackbox in 365.936792ms
 rendered prometheus in 371.855875ms
 rendered platform in 372.109916ms
--- import.values.git.commit.sh --
+-- import-values-git-commit.sh --
 git add . && git commit -m 'import values'
--- import.values.git.output.txt --
+-- import-values-git-output.txt --
 [main 52e90ea] import values
  2 files changed, 1815 insertions(+)
  create mode 100644 components/blackbox/values.cue
  create mode 100644 components/prometheus/values.cue
--- blackbox.common.config.header.sh --
-cat <<EOF > components/blackbox.cue
--- blackbox.common.config.body.cue --
-package holos
+-- mkdir-common-config.sh --
```

Comment:
Create the `config/prometheus` directory, and update the common config files so that:

- We use the correct path
- We use the `prometheus` package name (for `holos.example/config/prometheus`)